src/content/en/fundamentals/discovery-and-monetization/payment-request/index.md

```diff
@@ -754,8 +754,12 @@ do so, use `canMakePayment()`.
       request.show();
     }
 
-Note: This is only optional. Users can still add a new payment method in the
-Payment Request UI in general.
+Warning: In order to protect user privacy, calling <code>canMakePayment()</code>
+too often within a short time range causes [a "Quota exceeded"
```

Comment:
The details will evolve as we understand use cases better, so it's better to state our intent rather than details of implementation. However, if you insist on some more detail, we can state that QuotaExceededError goes away within 30 minutes or upon browser restart.  Let me know whether that's sufficient.